README.md

```diff
@@ -343,13 +298,15 @@ how to use the database to perform ML inference and processing.
 
 The following script launches a single database using the local launcher.
 
+[Experiment.create_database](https://www.craylabs.org/docs/api/smartsim_api.html#smartsim.experiment.Experiment.create_database)
```

Comment:
I can't comment on the line directly, but noticed it while reading the whole README.  I think the text "optional form a distributed database cluster" should be "optionally form a distributed database cluster"